package.json

```diff
@@ -16,10 +16,16 @@
   "bugs": {
     "url": "https://github.com/SU-SWS/decanter/issues"
   },
+  "dependencies": {
+    "bourbon": "^4.2.7",
```

Comment:
This version is what USA Gov Standards was using but it is behind and has much deprecated code in it. This will need to be updated.